scripts/eval/eval-prompt-change.py

```
@@ -0,0 +1,561 @@
+    before_results = comparison["before_results"]
+    after_results = comparison["after_results"]
+
+    # Criterion 1: no regression
+    no_regression = comparison["after_score"] >= comparison["before_score"]
+
+    # Criterion 2: at least one improvement (or no failures to begin with)
+    improvements = []
+    regressions = []
+    flaky_scenarios = []
+
+    for b, a in zip(before_results, after_results):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)